partnercenter/azext_partnercenter/clients/offer_client.py

```diff
@@ -0,0 +1,38 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
This was moved over from _client under operations, so it could be consumed by anything under the various operations subgroups cleanly